### DISCUSSION TITLE
Apollo Federation GraphQL Jar

### DISCUSSION THREAD

**Discussion by jkupcho** [Netflix/dgs-framework#17] [General]
The required Apollo Federation GraphQL jar (0.5.0) is only located within the JCenter repository. It might be worth noting that within the tutorial.

**Comment by paulbakker** [comment]
Thanks for catching this, I wasn't aware of it. Specially with jcenter being sunset (https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/) we'll need to figure this out soon. 

I'll look into it, and reach out to the owners of the Federation jar.

**Comment by setchy** [comment]
https://github.com/apollographql/federation-jvm/issues/91

**Reply by paulbakker** [reply]
Thanks for that @setchy. Way ahead of me 💯
